### ISSUE TITLE
Conflicts with  Power Mode

### ISSUE THREAD

**Issue by HarlemSquirrel** [pop-os/system76-power#317]
**Distribution (run `cat /etc/os-release`):**
 
NAME="Pop!_OS"
VERSION="21.10"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 21.10"
VERSION_ID="21.10"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=impish
UBUNTU_CODENAME=impish
LOGO=distributor-logo-pop-os
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

```
system76-power:
  Installed: 1.1.20~1646250081~21.10~39e6e94
  Candidate: 1.1.20~1646250081~21.10~39e6e94
  Version table:
 *** 1.1.20~1646250081~21.10~39e6e94 1001
       1001 http://apt.pop-os.org/release impish/main amd64 Packages
        100 /var/lib/dpkg/status
```

**Issue/Bug Description:**

Selecting "Battery Life" from the top right desktop menu always sets the Power Mode in the system settings to "Balanced Power".

![Screenshot from 2022-03-07 10-32-02](https://user-images.githubusercontent.com/6445815/157066182-ccd2d40b-9c2e-4f35-89fa-684d07366bab.png)

**Steps to reproduce (if you know):**

When selecting "Battery Life" from the top right desktop menu the Power Mode in the system settings is set to "Balanced Power" even if the setting was "Power Saver"

**Expected behavior:**

I expect Selecting "Battery Life" from the top right desktop menu to set the power mode to "Power Saver" or at least remember this setting if it was already set.

**Comment by mmstick** [comment]
Hold on. Do you still have this panel in GNOME Settings? This is for gnome-power-profiles and isn't compatible with system76-power. It's not there on Pop installs.

**Comment by HarlemSquirrel** [comment]
> Hold on. Do you still have this panel in GNOME Settings? This is for gnome-power-profiles and isn't compatible with system76-power. It's not there on Pop installs.

Yup I have this. Should I remove `gnome-power-profiles` package?

**Comment by mmstick** [comment]
Yes

**Comment by HarlemSquirrel** [comment]
> Yes

Oh I don't see this package actually, just `gnome-power-manager`

**Comment by techsy730** [comment]
You might want to consider marking in the debian/control file
`Conflicts: gnome-power-profiles gnome-power-manager`

That will cause apt (and dpkg) to loudly yell at you if you try to have both installed.

Though take care that removing the package doesn't cause like all of gnome to be uninstalled thanks to their tangled network of dependencies.
